tests/templates/kuttl/ldap-authentication/07-authcheck.yaml

```diff
@@ -0,0 +1,6 @@
+---
+apiVersion: kuttl.dev/v1beta1
+kind: TestStep
+timeout: 600
```

Comment:
This is a little much of a timeout for this check? Will extend the test feedback loop about 10minutes if anything fails here? Is this really necessary?

Reply:
I set it down to 60 seconds, just to make sure it's enough time.